entity-framework/core/what-is-new/release-planning.md

```diff
@@ -1,45 +1,122 @@
+### Patch releases
 
-3. **Does implementing this feature evolve the architecture of EF Core such that it moves us closer to implementing other features?** We tend to favor features that act as building blocks for other features. For instance, property bag entities can help us move towards many-to-many support, and entity constructors enabled our lazy loading support.
+Patch releases change only the "patch" part of the version.
+For example, EF Core 3.1.**1** is a release that patches issues found in EF Core 3.1.**0**.
 
-4. **Is the feature an extensibility point?** We tend to favor extensibility points over normal features because they enable developers to hook their own behaviors and compensate for any missing functionality.
+Patch releases are intended to fix critical customer bugs.
+This means patch releases do not include new features.
+API changes are not allowed in patch releases except in special circumstances.
 
-5. **What is the synergy of the feature when used in combination with other products?** We favor features that enable or significantly improve the experience of using EF Core with other products, such as .NET Core, the latest version of Visual Studio, Microsoft Azure, and so on.
+The bar to make a change in a patch release is very high.
+This is because it is critical that patch releases do not introduce new bugs.
+Therefore, the decision process emphasizes high value and low risk.
+
+We are more likely to patch an issue if:
+  * It is impacting multiple customers
+  * It is a regression from a previous release
+  * The failure causes data corruption
+
+We are less likely to patch an issue if:
+  * There are reasonable workarounds
+  * The fix has high risk of breaking something else
+  * The bug is in a corner-case
+
+This bar gradually rises through the lifetime of a [long-term support (LTS)](https://dotnet.microsoft.com/platform/support/policy/dotnet-core) release. This is because LTS releases emphasize stability.
+
+The final decision about whether or not to patch an issue is made by the .NET Directors at Microsoft.
+
+### Minor releases
+
+Minor releases change only the "minor" part of the version.
+For example, EF Core 3.**1**.0 is a release that improves on EF Core 3.**0**.0.
+
+Minor releases:
+* Are intended to improve the quality and features of the previous release
+* Typically contain bug fixes and new features
+* Do not include intentional breaking changes
+* Have a few prerelease previews pushed to NuGet
+
+### Major releases
+
+Major releases change the EF "major" version number.
+For example, EF Core **3**.0.0 is a major release that makes a big step forward over EF Core 2.2.x.
+
+Major releases:
+* Are intended to improve the quality and features of the previous release
+* Typically contain bug fixes and new features
+  * Some of the new features may be fundamental changes to the way EF Core works
+* Typically include intentional breaking changes
```

Comment:
I'd like to say we strive to avoid changes that will break applications and to minimize changes that will break providers and extensions, but I'm not sure that's historically accurate.

Reply:
We could say, "We may have been too aggressive with breaking changes in the past. Going forward, we will strive..."